src/app/components/ScrollablePromo/PromoList/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useContext } from 'react';
+import styled from '@emotion/styled';
+import { GEL_SPACING, GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_0_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import { arrayOf, shape, string, oneOfType, object } from 'prop-types';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Promo from '../Promo';
+
+const ScrollPromo = styled.ul`
+  display: flex;
+  list-style: none;
+  ${({ dir }) => `padding-${dir === 'ltr' ? 'left' : 'right'}: 0;`}
+  margin: 0;
+
+  overflow-x: scroll;
+  /* Avoid using smooth scrolling as it causes accessibility issues */
+  scroll-behavior: auto;
+  -webkit-overflow-scrolling: touch;
+
+  /* Hide scrollbar */
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+  &::-webkit-scrollbar {
+    display: none;
+  }
```

Comment:
I was confused at first about this emotion feature because it does not seem to work in this implementation. If I take away the vendor prefix the css in the specific browser simply does not work.

Reply:
This page does not suggest https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar is used in production. It looks like you could use [`scrollbar-width: none](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width) to achieve the same thing with a standardised attribute: https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width ?

Reply:
I am already using this attribute to target Firefox. Unfortunately scrollbar-width is only supported by Firefox forcing me to use vendor specific attributes.
<img width="1449" alt="Screenshot 2022-01-12 at 12 23 07" src="https://user-images.githubusercontent.com/90621252/149139691-85ce8988-caad-4f7c-a9a1-eacefe303abd.png">

Reply:
https://github.com/bbc/simorgh/pull/9729/files#diff-bd0d8f9bbdbe1b018012a5ccb6a313135ee6bfd9b79640b5b310dcc830e0e0feR24-R29